lib/handlers/connect.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const retimer = require('retimer')
-const pump = require('pump')
+const { pipeline } = require('stream')
```

Comment:
we should use the one from the readable-stream module instead (it's already on the dependency tree)